torchgeo/models/resnet.py

```diff
@@ -22,46 +22,16 @@
 _zhu_xlab_transforms = AugmentationSequential(
     K.Resize(256),
     K.CenterCrop(224),
-    K.Normalize(mean=0, std=10000),
+    K.Normalize(mean=torch.tensor(0), std=torch.tensor(10000)),
```

Comment:
Values must be torch tensors, not ints

Reply:
Fixed by https://github.com/kornia/kornia/pull/2269, but we should still use tensors or floats until the next release is the minimum release we support.